### PR TITLE
Re-implement indicator when running registered editor commands

### LINKED ISSUE #3954: Add back "PowerShell" text indicator when running a "Editor Command"
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all [issues](https://github.com/PowerShell/vscode-powershell/issues?q=is%3Aissue) to ensure it has not already been reported.

### Summary

In previous versions of the PowerShell extension, when I ran an Editor Command associated with PS1 file, it would show the text "PowerShell" in the bottom left corner.  When the command finished running, the PowerShell text would go away.  This was a nice indicator letting me when the command had finished, and I knew I could go look for the results.  In my case, my command (by use of Microsoft.VSCode_profile.ps1 with the script block registered as an Editor Command) signs the script, compiles the PS1 into an EXE format (for both .NET 4.8 and .NET 6.x formats), and then signs the executables.  This whole process takes a bit of time...long enough that the "PowerShell" text indicator "going away" really helped let me know when it was done.  With the current version, I have to wait "until I think it should be done" to check it.

image

As of version v2022.5.0, this feature no longer exists.

### Proposed Design

Please add the "PowerShell" text (or some other indicator) to the bottom (or other obvious location) of VSCode when an Editor Command is running and then hide it when the command finishes.  For an example of this, check out any previous version of the extension prior to v2022.5.0.  Hopefully this is an easy add, since it already worked in previous versions.

### PR DESCRIPTION
**Pull request by andschwa**
Resolves #3954 and uses the new language status bar item. Essentially just adds a spinner when the editor command is running.

### CODE REVIEW

**Review by SeeminglyScience — APPROVED** [review]
LGTM!

File: src/session.ts [hunk 1]
```diff
@@ -76,10 +77,6 @@ export const PowerShellVersionRequestType =
     new RequestType0<IPowerShellVersionDetails, void>(
         "powerShell/getVersion");
 
-export const RunspaceChangedEventType =
-    new NotificationType<IRunspaceDetails>(
-        "powerShell/runspaceChanged");
```
Comment (andschwa) [inline src/session.ts:81]: @SeeminglyScience this notification isn't wired up, and I wasn't planning to rewire it, so I deleted it. What do you think?

Comment (SeeminglyScience) [reply src/session.ts:81]: If it's not used I'm with ya for sure. It's in git history if we need it again 👍

**Comment by shangles** [comment]
I want to thank everyone involved in restoring this functionality to the PowerShell plug-in in Visual Studio Code.  I hope you all have a wonderful weekend.  Cheers!

**Pull request merged**